Bottom: c8f865e5ce4df28df3f2825d2cec17cb7a1efd80
Top:    6cd5bea13785a7f5cb98efc3a50651f9f39e3c8f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 14:37:56 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 4c0dcc3..5a686ac 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1612,6 +1612,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
    *
    */
 
+    fprintf (stderr, "Subtracting pin + via drills\n");
+    r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+    r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
   if (info.poly == NULL)
     {
       fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
